src/Synchronizer.js

```diff
@@ -88,11 +88,38 @@ module.exports = Class.extend({
          })
          .catch(function(err) {
             self._abortScanning = true;
-            console.log('ERROR: encountered an error while comparing tables', err, err.stack);
+            console.error('Encountered an error while comparing tables', err, err.stack);
          })
          .then(this._outputStats.bind(this));
    },
 
+   trackScanProgress: function(enabled, approxItems, f) {
```

Comment:
I don't see where `enabled` is being used?

Reply:
good catch, its not :)

Reply:
upon reflection, I'm now trying to figure out if the `enabled` makes sense here, if its not verbose, do these still make sense to show?  should the param just be removed from these methods?

Reply:
turns out it's not needed because you're not calling the tracking function unless it's non-verbose

If you were calling the tracking function, there'd be conflicts between the console and process writing I think